### PR TITLE
APP-1395: Whitesource integration

### PR DESCRIPTION
**Pull request by campidelli**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,8 +51,14 @@ before_script:
   - npm install -g gulp
 
 script:
-- "if [[ \"$TRAVIS_PULL_REQUEST\" = \"false\" && $TRAVIS_BRANCH =~ dev ]]; then mvn clean deploy -Pversioneye -Pcobertura -DtransitiveDependencies=true --settings settings.xml ; fi"
+- "if [[ \"$TRAVIS_PULL_REQUEST\" = \"false\" && $TRAVIS_BRANCH =~ dev ]]; then mvn clean deploy -Pversioneye,whitesource -Pcobertura -DtransitiveDependencies=true --settings settings.xml ; fi"
 - "if [[ \"$TRAVIS_PULL_REQUEST\" != \"false\" ]]; then mvn clean package -Pcobertura; fi"
 
+- npm install
```
Comment (robson-sanchez) [inline .travis.yml]: 'package.json' isn't in the root folder. Probably, it should be run in the 'src/main/webapp' folder.

**Review by robson-sanchez — APPROVED** [review]
LGTM

File: .travis.yml [hunk 2]
```diff
@@ -51,8 +51,14 @@ before_script:
   - npm install -g gulp
 
 script:
-- "if [[ \"$TRAVIS_PULL_REQUEST\" = \"false\" && $TRAVIS_BRANCH =~ dev ]]; then mvn clean deploy -Pversioneye -Pcobertura -DtransitiveDependencies=true --settings settings.xml ; fi"
+- "if [[ \"$TRAVIS_PULL_REQUEST\" = \"false\" && $TRAVIS_BRANCH =~ dev ]]; then mvn clean deploy -Pversioneye,whitesource -Pcobertura -DtransitiveDependencies=true --settings settings.xml ; fi"
 - "if [[ \"$TRAVIS_PULL_REQUEST\" != \"false\" ]]; then mvn clean package -Pcobertura; fi"
 
+- npm install --prefix ./src/main/webapp
+# Cannot run on external PRs due to https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions
+- "if [[ $TRAVIS_PULL_REQUEST -eq 'false' ]]; then npm run --prefix ./src/main/webapp whitesource; fi;"
```
Comment (maoo) [inline .travis.yml]: is `--prefix` needed here, since the `whitesource` script uses a relative path to resolve `node_modules` folder?

**Review by maoo — COMMENTED** [review]
LGTM 👍

**Pull request merged**